test/docfx.Test/lib/HtmlUtilityTest.cs

```diff
@@ -55,9 +56,16 @@ public void HtmlRemoveRerunCodepenIframes(string input, string output)
         [InlineData("<div><script></script></div>", "<div></div>")]
         public void HtmlStripTags(string input, string output)
         {
+            var allowlists = File.ReadAllText("data/validation/taxonomy-allowlists.json");
+            var taxo = OpsConfigAdapter.ConvertAllowedHTML(allowlists);
+            var config = new Config
+            {
+                AllowedHTML = taxo,
```

Comment:
Use the default `Config` for this test case. You can create a new test case to test `ConvertAllowedHtml` in `OpsConfigAdapterTest.cs`

Reply:
waiting IA team to import the taxonomy

Reply:
now data imported and test case added